lab/util/pytorch.py

```diff
@@ -28,3 +33,52 @@ def get_modules(configs: Configs):
             modules[k] = value
 
     return modules
+
+
+class BaseDataset(Dataset):
+    pass
+
+
+class CustomDataset(BaseDataset):
+    data: pd
+    y_cols: List
+    x_cols: List
+    transform: Callable
+    test_size: float = 0.0
```

Comment:
should `test_size` be an `int`?

Reply:
got it, it's a percentage. Can we use a separate name like `test_fraction` for this and keep `test_size` separate